packages/shadydom/src/patch-prototypes.js

```diff
@@ -196,7 +199,12 @@ export const patchElementProto = (proto, isVanillaElement = false) => {
     copyAllImplementedNativeProperties(proto);
   }
   proto[PROTO_IS_PATCHED] = true;
-  applyAllHTMLElementProtoPatches(proto);
+  allHTMLElementOnDemandPatches.forEach((patches) => {
```

Comment:
Question (can't recall): We may need to on-demand non-element (TextNode, CommentNode, etc.); do we just patch in all of the HTMLElement patches onto those nodes and call it good enough? Or is the node type being patched discriminated somewhere?